backend/server/src/db/migrations/4-events.sql

```diff
@@ -0,0 +1,20 @@
+-- TODO Use the proper settings for this
+--   We should not be able to tamper with it any more than we can with an internal sequence.
+--   What's its datatype?!
+create sequence internal_event_ids;
+
+-- TODO Can we somehow make this immutable?!
+--   Maybe using triggers ...?
+insert into __keys (entity) values ('events');
```

Comment:
What does "immutable" mean in this context?

Reply:
What I mean by this is: Can we somehow prevent that the key that is generated by this `insert` is ever changed. Or at least make it hard to do it by accident.

Of course there might be reasons where people might want to change it, but that's a very disruptive thing to want to do, because that breaks the ID generation, of course, in the sense that the IDs generated after the change might now collide with the old ones.

So yeah, dunno, this TODO will probably just disappear in the final PR. x)

Reply:
Ah okay. Yeah I agree, that would be nice to do.

Reply:
So yeah, as predicted, this just went away for now. Making this and a few other things 100% air tight might get hairy, and I would like to avoid that at least for now. :thinking: